cypress/integration/portal-dashboard/portal-dashboard-question-details.spec.js

```diff
@@ -151,7 +151,7 @@ context("Portal Dashboard Question Details Panel", () => {
       cy.get('[data-cy="answer-image"]').should('be.visible');
     });
     it('verify image lightbox of image answer', ()=>{
-      cy.get('[data-cy="magnify-answer"]').should('be.visible').click();
+      cy.get('[data-cy="magnify-answer"]').should('be.visible').eq(0).click();
```

Comment:
cy.get('[data-cy="question-overlay"] [data-cy="magnify-answer"]') instead of .eq(0).

Reply:
Fixed.